### ISSUE TITLE
[SOLVED] No Enable button on ImgToImg and Extras tabs

### ISSUE THREAD

**Issue by Nampo** [Gourieff/sd-webui-reactor#314]
### First, confirm

- [X] I have read the [instruction](https://github.com/Gourieff/sd-webui-reactor/blob/main/README.md) carefully
- [X] I have searched the existing issues
- [X] I have updated the extension to the latest version

### What happened?

The Enable button is absent when in ImgToImg and Extras tabs.
In TextToImage there is a checkbox on the "ReActor" title, but it's not present in the other tabs.
The last commit that has the Enable checkbox in ImgToImg is ed66d918190a296c745265a01c090e3e3dbc2cde on 19/12/2023
Starting from d8e300b202e54199d3dc63468a89f16a048fcfba it's missing.

### Steps to reproduce the problem

1. launch A1111
2. go to ImgToImg or Extras
3. see that ReActor can't be enabled

### Sysinfo

WebUI: Automatic1111 1.7.0
OS: Windows 10 Pro
Browser: Firefox - latest

### Relevant console log

```Shell
venv "G:\SD\AUTOMATIC1111\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: v1.7.0
Commit hash: cf2772fab0af5573da775e7437e6acdca424f26e
Installing requirements for Batch Face Swap
Installing requirements for Face Editor
Faceswaplab : Use GPU requirements
Checking faceswaplab requirements
Install protobuf>=3.20.2
Installing sd-webui-faceswaplab requirement: protobuf>=3.20.2
1.476559400005499
ReActor preheating... Device: CUDA
Check mmcv version...
Your mmcv version 2.0.1 may not work mmyolo.
Please install mmcv version 2.0.0 manually or uninstall mmcv and restart UI again to install mmcv 2.0.0
Check mmengine version...
Your mmengine version 0.10.2 may not work windows...
Please install mmengine 0.8.5 manually or install windows enabled un-official patched version of bitsandbytes with
Launching Web UI with arguments: --xformers --medvram
Civitai Helper: Get Custom Model Folder
Tag Autocomplete: Could not locate model-keyword extension, Lora trigger word completion will be limited to those added through the extra networks menu.
[-] ADetailer initialized. version: 24.1.0, num models: 20
[AddNet] Updating model hashes...
0it [00:00, ?it/s]
[AddNet] Updating model hashes...
0it [00:00, ?it/s]
ControlNet preprocessor location: G:\SD\AUTOMATIC1111\stable-diffusion-webui\extensions\sd-webui-controlnet\annotator\downloads
2024-01-07 05:38:11,728 - ControlNet - INFO - ControlNet v1.1.427
2024-01-07 05:38:11,790 - ControlNet - INFO - ControlNet v1.1.427
05:38:12 - ReActor - STATUS - Running v0.6.0-a2
[text2prompt] Following databases are available:
    all-mpnet-base-v2 : danbooru_strict
Loading weights [b370189733] from G:\SD\AUTOMATIC1111\stable-diffusion-webui\models\Stable-diffusion\juggernaut_aftermath-inpainting.safetensors
Creating model from config: G:\SD\AUTOMATIC1111\stable-diffusion-webui\models\Stable-diffusion\juggernaut_aftermath-inpainting.yaml
 Total 8 mmdet, 0 yolo and 3 mediapipe models.
SUCCESS - success to load config for G:\SD\AUTOMATIC1111\stable-diffusion-webui\models\mmdet\bbox\face_yolov8n.pth!
 You can enable model validity tester in the Settings-> μ DDetailer.
SUCCESS - success to load config for G:\SD\AUTOMATIC1111\stable-diffusion-webui\models\mmdet\bbox\face_yolov8s.pth!
SUCCESS - success to load config for G:\SD\AUTOMATIC1111\stable-diffusion-webui\models\mmdet\bbox\hand_yolov8n.pth!
SUCCESS - success to load config for G:\SD\AUTOMATIC1111\stable-diffusion-webui\models\mmdet\bbox\hand_yolov8s.pth!
SUCCESS - success to load config for G:\SD\AUTOMATIC1111\stable-diffusion-webui\models\mmdet\bbox\mmdet_anime-face_yolov3.pth!
SUCCESS - success to load config for G:\SD\AUTOMATIC1111\stable-diffusion-webui\models\mmdet\segm\mmdet_dd-person_mask2former.pth!
SUCCESS - success to load config for G:\SD\AUTOMATIC1111\stable-diffusion-webui\models\mmdet\segm\yolov5_ins_n.pth!
SUCCESS - success to load config for G:\SD\AUTOMATIC1111\stable-diffusion-webui\models\mmdet\segm\yolov5_ins_s.pth!
 Total 8 valid mmdet configs are found.
 You can disable validity tester in the Settings-> μ DDetailer.
Check config files...
Done
Applying attention optimization: xformers... done.
Model loaded in 3.6s (load weights from disk: 0.2s, create model: 0.3s, apply weights to model: 1.4s, apply half(): 0.7s, calculate empty prompt: 1.0s).
loading.. cn_modole
 - ControlNet extension sd-webui-controlnet found
 - default µ DDetailer model= bbox/face_yolov8n.pth [882ebbb6]
 - default µ DDetailer model= bbox/face_yolov8n.pth [882ebbb6]
Civitai Helper: Set Proxy:
*Deforum ControlNet support: enabled*
[text2prompt] Loading database with name "all-mpnet-base-v2 : danbooru_strict"...
[text2prompt] Database loaded
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 35.5s (prepare environment: 18.2s, import torch: 2.1s, import gradio: 0.6s, setup paths: 0.6s, initialize shared: 0.3s, other imports: 0.5s, load scripts: 6.3s, scripts before_ui_callback: 1.2s, create ui: 4.6s, gradio launch: 0.7s, app_started_callback: 0.3s).
```

**Comment by Gourieff** [comment]
Just click it to open and it becomes enabled
It works the same way as other A1111 extensions since 1.7.0 version

![изображение](https://github.com/Gourieff/sd-webui-reactor/assets/85128026/79242f78-b3c5-47e1-918f-7965044f7f8b)

---

![изображение](https://github.com/Gourieff/sd-webui-reactor/assets/85128026/6dcf5d22-c79e-4a2b-bdcb-2c7ef763a9fb)

**Comment by atxoxx** [comment]
For me strangely it doesn't work... 
I'm on latest automatic 1111 update + latest reactor version 
even when reactore is extended it doesn't activate
So I manually edited the .py files from before ui change and I have now the enable button, and reactor process the image

**Comment by Gourieff** [comment]
Strange... What browser do you use?

**Comment by atxoxx** [comment]
I use edge 
So I reinstalled another instance of automatic 1111, and tick box is here 
so an extension must be messing with this UI 
(I should check better before posting ^^', sorry for the incoveniance) 
I'm gonna check the addon that cause this issue and report

**Comment by Nampo** [comment]
I use Firefox, and the checkbox at the left of the "Reactor" label does not show up, no matter if I click it open or not.

**Comment by Gourieff** [comment]
Update your A1111 to the 1.7.0 version

**Comment by Nampo** [comment]
I'm using A1111 Version: v1.7.0
No checkbox in the img2img tab:
![image](https://github.com/Gourieff/sd-webui-reactor/assets/11336956/c5f5e694-4a94-49fe-8c6e-6a9fc1bc5ebc)

**Comment by Nampo** [comment]
I figured it out.
I tried removing all the other extensions and re-adding them in small batches, and I found out that "μ Detection Detailer"(https://github.com/wkpark/uddetailer) is causing the issue with ReActor:
![image](https://github.com/Gourieff/sd-webui-reactor/assets/11336956/e468108a-0c50-4175-a386-1c78e1459260)
I'm going to remove that one, but you might want to either investigate the incompatibility (since it acts on your extension, not their, so you are going to be bothered by people), or put an incompatibility warning in the readme.
Thanks for the support anyway guys

**Comment by Gourieff** [comment]
According to the code "μ Detection Detailer" overrides built-in A1111's "enable-disable on open-close" Accordion behavior, this is not good
This will cause conflicts not only with ReActor but also with other extensions which can use A1111's built-in accordion's "enable-disable on open-close" function
